md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/FileAPI.java

```diff
@@ -60,11 +61,11 @@ RestWrapper get(
             File file = new File();
             file.setBatchId(batchId);
             file.setCreationTS(DateConverter.stringToDate(file.getTableCreationTS()));
-            //file = s.selectOne("call_procedures.GetFile", file);
             file = fileDAO.getFile(file);
             restWrapper = new RestWrapper(file, RestWrapper.OK);
             LOGGER.info("Record with ID:" + batchId + " selected from File by User:" + principal.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Record with ID:" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1192)